tests/integrations/aws_lambda/test_aws.py

```diff
@@ -362,3 +383,132 @@ def test_handler(event, context):
     assert envelope["contexts"]["trace"]["op"] == "serverless.function"
     assert envelope["transaction"].startswith("test_function_")
     assert envelope["transaction"] in envelope["request"]["url"]
+
+
+@pytest.mark.parametrize(
+    "aws_event, has_request_data, batch_size",
+    [
```

Comment:
Let's trim down the amount of parametrizations here unless this truly tests edgecases. Each of this will add 5 seconds to overall test time.

Reply:
Okay, pulled ~80 sec worth of options.